test/basic.html

```diff
@@ -36,36 +13,252 @@
 <body>
 	<test-fixture id="cosmoz-grouped-list-fixture">
 		<template>
-			<cosmoz-grouped-list></cosmoz-grouped-list>
+			<cosmoz-grouped-list>
+				<cosmoz-grouped-list-template id="itemTemplate">
+					<template>
+						<div class="item-template" style="border-bottom: 1px solid grey;">
+							<div>
+								ID:<span class="item-id">{{ item.id }}</span>
+								NAME:<span class="item-name">{{ item.name }}</span>
+								VALUE:<span class="item-value">{{ item.value }}</span>
+								<div>
+						</div>
+					</template>
+				</cosmoz-grouped-list-template>
+				<cosmoz-grouped-list-template id="groupTemplate">
+					<template>
+						<div class="group-template">
+							NAME:<span class="item-name">{{ item.name }}</span>
+							VALUE: <span class="item-value">{{ item.value }}</span>
+							SELECTED: <span class="item-selected">{{ selected }}</span>
+						<div>
+					</template>
+				</cosmoz-grouped-list-template>
+			</cosmoz-grouped-list>
 		</template>
 	</test-fixture>
 
 	<script>
-	/*global WCT, suite, fixture, setup, test, Polymer, assert, logStore, errorStore */
 	(function () {
 		'use strict';
 
 		suite('<cosmoz-grouped-list>', function () {
-			var myElement;
+			var element
```

Comment:
No eslint rule for that. Will add [comma-style](http://eslint.org/docs/rules/comma-style) rule to enforce it.